mttl/models/expert_model.py

```diff
@@ -693,3 +680,22 @@ def training_step(self, batch, _):
         for i, pg in enumerate(self.optimizers().optimizer.param_groups):
             self.log(f"train/lr_{i}", pg["lr"])
         return total_loss
+
+    def as_expert(self):
```

Comment:
how does `as_expert` work for multiple experts ?

Reply:
ups, this should not be here. Addressed